code-analysis/crates/codegen/parser/src/chumsky/typescript_lib_boilerplate.rs

```diff
@@ -63,19 +63,17 @@ pub fn lex_head() -> TokenStream {
 
             #[napi(getter)]
             pub fn start(&self) -> usize {
-                if let Node::Chars(range) = self.0.as_ref() {
-                    range.start
-                } else {
-                    unreachable!("Expected a chars node")
+                match self.0.as_ref() {
+                    Node::Chars(range) => range.start,
+                    _  => unreachable!()
```

Comment:
I wonder why the error messages were removed in this PR?
Wouldn't that make the errors less readable?

Reply:
It really is unreachable by construction, but Rust isn't powerful enought